db/models/getArticleById.model.js

```diff
@@ -0,0 +1,7 @@
+const db = require("../connection");
+
+exports.findArticleById = (id) => {
+    return db.query(`SELECT * FROM articles WHERE article_id = ${id}`).then(({rows}) => {
+        return rows;
+    })
```

Comment:
This leaves you open to sql injections. Look at parameterized queries

https://notes.northcoders.com/courses/js-back-end/node-postgres#parameterized-queries